### ISSUE TITLE
TinyMCE presents a "tab trap" when navigating a page on Chrome

### ISSUE THREAD

**Issue by dumptruckman** [tinymce/tinymce#4459]
This appears to be a bug related to accessibility. When a TinyMCE editor is present on a page, tab navigating through the page will be stopped by navigating into the text area of the editor. Once focus is in the editor, it appears to be impossible to keyboard navigate past it. Shift-tab seems to escape in the opposite direction which just means navigation cannot continue past the editor.

It should be possible to navigate past the editor using the keyboard. According to WCAG 2.1.2, having some sort of escape behavior (such as a different keyboard shortcut) would be sufficient.

This is happening in Chrome but is not an issue in Edge and Firefox. We are using the latest TinyMCE 4.

**Comment by dumptruckman** [comment]
Any ideas where I could start looking into this so maybe I can PR a fix?

**Comment by dumptruckman** [comment]
It seems like this isn't always a problem except under maybe certain configurations. However, using nerdefocus I've identified a possibly related issue. Using the [full featured demo example](https://www.tiny.cloud/docs/demo/full-featured/) I have found that the focus does not properly transfer to the text area. Tabbing through the page, the focus stays on the element just before the editor when control has clearly transferred to the editor (which is indicated by being able to type in it). In this demo, the tab trap does not exist.

I've also found that in our own usage, using version 4.7.11 does not produce the tab trap, though this improper focus still exists. In version 4.7.12 and 4.7.13, the tab trap does exist. Unfortunately, I have not figured out how to produce this tab trap in a standalone manner. Until then I'm working off the assumption that this improper focus is related.
